muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/hooks/useNotes.ts

```diff
@@ -18,12 +23,12 @@ export const useOnGoingNotes = (
   displayNotification: DisplayNotificationTriggerType
 ) => {
   const [notes, setNotes] = React.useState(<Note[]>[]);
-  const { userId, role } = status;
+  const { userId, roles } = status;
   const { t } = useTranslation("tasks");
 
   React.useEffect(() => {
     // This is for students only hook, if you call it as someone else, no loading should happen
-    if (role !== "STUDENT") {
+    if (roles ? !roles.includes(Role.Student) : false) {
```

Comment:
Unnecessary null/undefined check as roles is always empty list by default. Can use includes checking straight